debug.log

```diff
@@ -0,0 +1,12 @@
+[0910/135806.354:ERROR:process_reader_win.cc(123)] NtOpenThread: {Access Denied} A process has requested access to an object, but has not been granted those access rights. (0xc0000022)
+[0910/135806.370:ERROR:exception_snapshot_win.cc(99)] thread ID 11316 not found in process
+[0910/135806.354:ERROR:process_reader_win.cc(123)] NtOpenThread: {Access Denied} A process has requested access to an object, but has not been granted those access rights. (0xc0000022)
+[0910/135806.370:ERROR:exception_snapshot_win.cc(99)] thread ID 3376 not found in process
```

Comment:
i don't think this file should be checkin-in , please check with poll

Reply:
Removed